tests/fixtures/issues/issue-436.php

```diff
@@ -4930,59 +4877,6 @@
         'lite' => true,
         'standard' => true,
     ],
-    'issue-436-CR' => [
-        'ua' => 'Mozilla/5.0 (iPhone; CPU iPhone OS 10_10 like Mac OS X) AppleWebKit/600.1.4 (KHTML, like Gecko) Version/8.0 Mobile/12B401 Safari/600.1.4',
```

Comment:
Ah, I see.

Should the `iOS_B_8_0` platform be added to every case that there is `iOS_A_8_0` then?